acai-ios/CollectionViewCells/RadioSelectionCollectionViewCell.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  RadioSelectionCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class RadioSelectionCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: View vars
+    var titleLabel: UILabel!
+    var priceLabel: UILabel!
+    var imageView: UIImageView!
+    var radioCircle: UIView!
+    var radioFill: UIView!
+    var line: UIView!
```

Comment:
should these be alphabetized? @kevinchan159 @drewsdunne

Reply:
we don't really enforce alphabetization too much but it would be nice to have